onnx/defs/math/defs.cc

```diff
@@ -227,14 +227,24 @@ ONNX_OPERATOR_SET_SCHEMA(
 
 static const char* Relu_ver6_doc = R"DOC(
 Relu takes one input data (Tensor<T>) and produces one output data
-(Tensor<T>) where the rectified linear function, y = max(0, x), is applied to
-the tensor elementwise.
+(Tensor<T>). For each element x in input, if x > threshold then y = x;
+else y = value. By default the value and threshold all equal to 0.
 )DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     Relu,
     6,
     OpSchema()
+        .Attr(
```

Comment:
we're having a design issue here.

The attribute "threshold" type should be same as input type, while the attribute "value" type should be the same as output type.

Before we unifying the type system for attribute, input and output, let's set the attribute type as "Tensor", which looks better than current setting, though it's not perfect.

Reply:
@linkerzhang  btw since in relu, the output type should be the same to the input type, that means "x" "y" "value" "threshold" should all share one type.